vaccine_feed_ingest/runners/co/colorado_gov/parse.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import json
+import logging
+import pathlib
+import sys
+
+from lxml import etree
+from pykml import parser
+
+logger = logging.getLogger("co/colorado_gov/parse.py")
+
+
+def parse_point(element):
+    coords = element.coordinates.text.strip().split(",")
+    return {"long": coords[1], "lat": coords[0]}
```

Comment:
It looks to me like this might still parse to out of bounds values. 

Should it be something like this? 
```suggestion
    return {"long": coords[0], "lat": coords[1]}
```

Reply:
🤦 oof, I switched the order but not the indices. I'll re-verify this

Reply:
Ok, so this was right all along. In my comment above where I checked coordinates, I used the current parsed values. As it parses now, with the order `lat,long` the following values are correct as currently parsed.
- `{"long": "-108.5699514", "lat": "39.0670191"}`
- `{"long": "-104.8918168", "lat": "39.8056202"}`